cmd/cerbosctl/inspect/policies.go

```diff
@@ -55,7 +60,7 @@ func (c *PoliciesCmd) Run(k *kong.Kong, cctx *client.Context) error {
 
 	tw := printer.NewTableWriter(k.Stdout)
 	if !c.Format.NoHeaders {
-		tw.SetHeader([]string{"POLICY ID", "ACTIONS", "VARIABLES"})
+		tw.SetHeader([]string{"POLICY ID", "POLICY", "ACTIONS", "VARIABLES"})
```

Comment:
The `POLICY` column is redundant and just takes up space so, let's not display it.